src/app/public/modules/vertical-tabset/vertical-tabset.service.ts

```diff
@@ -34,8 +35,10 @@ export class SkyVerticalTabsetService {
   public tabAdded: Subject<SkyVerticalTabComponent> = new Subject();
   public indexChanged: BehaviorSubject<number> = new BehaviorSubject(undefined);
   public switchingMobile: Subject<boolean> = new Subject();
+  public maintainTabContent: boolean = false;
 
-  public animationVisibleState: string;
+  public animationTabsVisibleState: string;
+  public animationContentVisibleState: string;
```

Comment:
Alphabetical ordering of properties.

Reply:
Done. Also fixed legacy issues with that in this file.